src/dart.rs

```diff
@@ -54,7 +54,7 @@ impl DartGenerator {
             ffi.Pointer<T> _lookupDartSymbol<T extends ffi.NativeType>(String symbol) {
                 final ffi.Pointer<_DartApi> api = ffi.NativeApi.initializeApiDLData.cast();
                 final ffi.Pointer<_DartApiEntry> functions = api.ref.functions;
-                for (var i = 0; i < 100; i++) {
+                for (var i = 0; i < 10000; i++) {
```

Comment:
is there any reasonable limit we need to observe here? Like, we break whenever we find it, right? And we break as soon as no more function is found. So, why limit to 1000 (or any other specific value)? And there is literally only two uses of this fn in here. So, I propose to let it run to the max boundary of int:
```suggestion
               final maxInt = double.maxFinite.toInt();
                for (var i = 0; i < maxInt; i++) {
```

Reply:
Yeah, you are correct.